src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -114,7 +117,30 @@ public void setUpdateDate(Timestamp updateDate) {
     this.updateDate = updateDate;
   }
 
-  /**
+  public Integer getCountUsers() {
+    List<String> emails = null;
+    if (Objects.nonNull(data)) {
+      ArrayList<Collaborator> collabs = new ArrayList<>();
+      if (Objects.nonNull(data.getInternalCollaborators())) {
+        collabs.addAll(data.getInternalCollaborators());
+      }
+      if (Objects.nonNull(data.getLabCollaborators())) {
+        collabs.addAll(data.getLabCollaborators());
+      }
+      emails = collabs.stream().map(collaborator -> Objects.requireNonNull(collaborator).getEmail()).collect(Collectors.toList());
+
+      if (Objects.nonNull(data.getAcademicEmail())) {
+        emails.add(data.getAcademicEmail());
+      }
+      if (Objects.nonNull(data.getPiEmail())) {
+        emails.add(data.getPiEmail());
+      }
+      emails = emails.stream().map(String::toLowerCase).distinct().collect(Collectors.toList());
+    }
+    return Objects.nonNull(emails) ? emails.size() : 0;
+  }
```

Comment:
This method really should be in the metrics class. In general, we don't need to know this value for a DAR. The only reason for it is for metrics, so let's keep that code close to where it is used.